lib/src/remote_devtools_middleware.dart

```diff
@@ -32,11 +32,13 @@ class RemoteDevToolsMiddleware extends MiddlewareClass {
   String _channel;
   RemoteDevToolsStatus status = RemoteDevToolsStatus.notConnected;
 
+  ActionDecoder actionDecoder;
   ActionEncoder actionEncoder;
   StateEncoder stateEncoder;
 
   RemoteDevToolsMiddleware(this._host,
-      {this.actionEncoder = const JsonActionEncoder(),
+      {this.actionDecoder = const NOPActionDecoder(),
+      this.actionEncoder = const JsonActionEncoder(),
       this.stateEncoder = const JsonStateEncoder(),
       this.socket}) {
```

Comment:
If you add a trailing comma after this.socket it will improve formatting. https://flutter.io/docs/development/tools/formatting#using-trailing-commas

Reply:
I'll see what the VSCode flutter plugin does with the formatting. My styleguide is "use vscode's dart plugin" ;)